superset/migrations/versions/3dda56f1c4c6_migrate_num_period_compare_and_period_.py

```diff
@@ -50,6 +51,12 @@ class Slice(Base):
     'hour': 'PT1H',
     'day': 'P1D',
     'week': 'P1W',
+    'week_ending_saturday': 'P1W',
+    'week_start_sunday': 'P1W',
+    'week_start_monday': 'P1W',
+    'week_starting_sunday': 'P1W',
+    'P1W/1970-01-03T00:00:00Z': 'P1W',
```

Comment:
Note I'm unsure why these `P1W/` and `/P1W` grains persist in the form-data as opposed to `week_ending_saturday` and `week_ending_sunday` respectively.